GliaWidgets/Lib/ViewController/NavigationController/NavigationController.swift

```diff
@@ -1,3 +1,47 @@
 import UIKit
 
-class NavigationController: UINavigationController {}
+class NavigationController: UINavigationController {
+    private let transitionDuration: CFTimeInterval = 0.3
+
+    override func pushViewController(_ viewController: UIViewController, animated: Bool) {
+        if animated {
+            let transition = CATransition()
+            transition.duration = transitionDuration
+            transition.timingFunction = CAMediaTimingFunction(name: .easeInEaseOut)
+            transition.type = .push
+            transition.subtype = .fromRight
+            view.layer.add(transition, forKey: nil)
+        }
+        
+        super.pushViewController(viewController, animated: false)
```

Comment:
probably we could pass animated and don't pass manually `false` for the animated parameter. (the same for other methods in this class)
```suggestion
        super.pushViewController(viewController, animated: animated)
```

Reply:
oh, good catch. You are right. Thank you for clarification.